docs/intro/overview.rst

```diff
@@ -13,172 +13,112 @@ precisely, `web scraping`_), it can also be used to extract data using APIs
 (such as `Amazon Associates Web Services`_) or as a general purpose web
 crawler.
 
-The purpose of this document is to introduce you to the concepts behind Scrapy
-so you can get an idea of how it works and decide if Scrapy is what you need.
 
-When you're ready to start a project, you can :ref:`start with the tutorial
-<intro-tutorial>`.
+Walk-through of an example spider
+=================================
 
-Pick a website
-==============
+In order to show you what Scrapy brings to the table, we'll walk you
+through an example of a Scrapy Spider using the simplest way to run a spider.
 
-So you need to extract some information from a website, but the website doesn't
-provide any API or mechanism to access that info programmatically.  Scrapy can
-help you extract that information.
+Once you're ready to dive in more, you can :ref:`follow the tutorial
+and build a full-blown Scrapy project <intro-tutorial>`.
```

Comment:
I think this note can be moved to the end - it is unclear if users should continue reading the overview, of if they should go to the tutorial. It seems the reason you've put it here is that in addition to 'scrapy runspider' there is 'scrapy crawl' with a full-blown project support, and you wanted to mention it. We can add project support to a list of Scrapy advantages - Scrapy helps to organize the code, so that projects with tens and hundreds of spiders are still manageable.